src/fpu/fpu_instructions.h

```diff
@@ -266,10 +279,19 @@ static void FPU_FST_I32(PhysPt addr) {
 	mem_writed(addr,(val < 2147483648.0 && val >= -2147483648.0)?static_cast<int32_t>(val):0x80000000);
 }
 
-static void FPU_FST_I64(PhysPt addr) {
-	double val = FROUND(fpu.regs[TOP].d);
+static void FPU_FST_I64(PhysPt addr)
+{
 	FPU_Reg blah;
-	blah.ll = (val < 9223372036854775808.0 && val >= -9223372036854775808.0)?static_cast<int64_t>(val):LONGTYPE(0x8000000000000000);
+	// If the value was loaded with the FILD/FIST 64-bit memcpy trick,
```

Comment:
`blah`... 🤦🏻

Reply:
`blah`, being a potential placeholder world in English, was recommended above less flexible interjectives like `argh`, `ugh`, `grrr`, and `doh` 🤦